src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -570,11 +587,12 @@ def _update_mons(self, num, hosts=None):
         def split_host(host):
```

Comment:
And remove this copy

Reply:
there are 2 places that use this version, and one place that uses a different version of split_host... wasn't sure if it belonged at top level

Reply:
Looks like there is a pattern that this is going to get used more often